### PR TITLE
Changed accordion analytics tracking

### PR DESCRIPTION
**Pull request by zagi25**
Changes:

- Changed the `daa-ll` attribute of the accordion button to the format that was requested in the ticket.

Resolves: [MWPW-135027](https://jira.corp.adobe.com/browse/MWPW-135027)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/ratko/accordion
- After: https://accordion-analytics--milo--zagi25.hlx.page/drafts/ratko/accordion

### CODE REVIEW

**Review by mokimo — APPROVED** [review]

**Comment by mokimo** [comment]
Please ensure to add a PR label and the linter should be running through as well! (Even if you didn't necessarily touch ESLint errors above in the code, it's good to be a good pathfinder and leave the code better than u found it!)

**Review by overmyheadandbody — APPROVED** [review]
Agree with Okan on fixing ESLint issues when they're noticed

File: libs/blocks/accordion/accordion.js [hunk 1]
```diff
@@ -48,13 +47,15 @@ function displayMedia(displayArea, el, dd, i, expanded) {
   );
 }
 
-function handleClick(el, dd, num, id) {
+function handleClick(el, dd, num) {
   const expanded = el.getAttribute('aria-expanded') === 'true';
   if (expanded) {
     el.setAttribute('aria-expanded', 'false');
+    el.setAttribute('daa-ll', el.getAttribute('daa-ll').replace(/\|close\|/, '|open|'));
```
Comment (narcis-radu) [inline libs/blocks/accordion/accordion.js:54]: You should be able to cache the `daa-ll` value before line 52.
```js
const analyticsValue = el.getAttribute('daa-ll');
if (expanded) {
  // do your logic
} else {
  // do your logic
}
```

**Review by narcis-radu — APPROVED** [review]

**Comment by zagi25** [comment]
Hi @mokimo, @narcis-radu, @overmyheadandbody, can we please put this on hold for the moment until the conversation on the ticket [MWPW-135027](https://jira.corp.adobe.com/browse/MWPW-135027) is resolved? Maybe this won't be needed in the future, or perhaps some additional changes are required. Thank you.

**Pull request merged**